README_GPDB_ru.md

```diff
@@ -0,0 +1,207 @@
+# 1. Поддерживаемые версии Greenplum
+
+На данный момент поддерживается только Greenplum 6.
+
+# 2. Компиляция pgBackRest
+
+- Установить зависимости
+
+для CentOS 7:
+```
+yum install gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel lz4-devel libyaml-devel
```

Comment:
When checking today on a "new and shiny" installation of CentOS 7, git wasn't there by default. So I think we need to add 'git' into the list of dependencies.